app/src/main.ts

```diff
@@ -30,6 +31,16 @@ if (require('electron-squirrel-startup')) {
 
 const appArgs = JSON.parse(fs.readFileSync(APP_ARGS_FILE_PATH, 'utf8'));
 
+// Do this relatively early so that we can start storid appData with the app
+if (appArgs.portable) {
+  log.debug(
```

Comment:
This log isn't visible. And I realize that none of the `log.` logs in `app` are visible (neither in the terminal, nor in the app's devtools/console), because we do import `log` but we never set the level.

Since there are many instances of the problem (all in `app/main.ts` and `app/mainWindow.ts`), I suggest we let it be in this PR, but a separate PR could address this, by

1. Looking at argv and setting `log`'s level, to `info` or to `debug` if a `--debug` or `--verbose` arg is passed.
2. Converting `console.`... calls in `app` to use `log` (14 hits across 6 files in `app`)

Reply:
> This log isn't visible. And I realize that none of the `log.` logs in `app` are visible (neither in the terminal, nor in the app's devtools/console), because we do import `log` but we never set the level.
> 
> Since there are many instances of the problem (all in `app/main.ts` and `app/mainWindow.ts`), I suggest we let it be in this PR, but a separate PR could address this, by
> 
> 1. Looking at argv and setting `log`'s level, to `info` or to `debug` if a `--debug` or `--verbose` arg is passed.
> 2. Converting `console.`... calls in `app` to use `log` (14 hits across 6 files in `app`)

No problem. I'll get a separate PR going.

Reply:
#1172